index.html

```diff
@@ -6,8 +6,20 @@
     <title>Resque_a_pet</title>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/fonts.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/main.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/heading.css" type="text/css"/>
+    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
 </head>
 <body>
-    
+ <div class="container">
+  <header>
+     <img class="logo-header" src="assets/images/logo.png"/>
+     <img class="photo-header" src="assets/images/the-dog-white-and-brown.jpeg"/>   
+     <i class="fb"></i>
```

Comment:
Co to jest fb? Facebook. Ale co facebook? Link, logo, ikona, opis?
Zrób wrapper dla ikon. 
```<div class="header-social-media-icons">```
ikony więc zrobiłbym tak:

```
<i class="header-social-media fb"></i>
<i class="header-social-media tw"></i>
<i class="header-social-media ig"></i>
```
Pozwala Ci to na:
- zrobienie jednej klasy dla wszystkich ikon (ustawisz tylko raz wielkość, marginesy, kolor i tak dalej)
- modyfikowanie dodatkowe każdej ikony z osobna (możesz np. zrobić fb:hover - niebieski, tw:hover - niebieski jasny, tw:hover - pomarańczowy)

Poza tym nie podpiąłeś font awesome. 
Nie mergujemy nic, co nie jest gotowe. 
Najpierw podepnij fontawesome, Jak sobie nie poradzisz (trochę inaczej działa niż fontello - musisz się uczyć praktycznie każdej rzeczy od nowa), to pisz.

Reply:
Poprawione